docs/resources/dws_cluster_v1.md

```diff
@@ -68,7 +68,6 @@ The following arguments are supported:
     - Start with a lowercase letter or an underscore.
     - Contain 1 to 63 characters.
     - Cannot be a keyword of the DWS database.
-  Changing this will create a new resource.
```

Comment:
Why remove this line ?

Reply:
not remove, display error

Reply:
![image](https://github.com/FlexibleEngineCloud/terraform-provider-flexibleengine/assets/8384921/8a433ea7-06ed-405e-8790-a6217e653be6)

Where is the error? Please keep it.